freqtrade/configuration.py

```diff
@@ -375,22 +376,41 @@ def get_config(self) -> Dict[str, Any]:
 
         return self.config
 
-    def check_exchange(self, config: Dict[str, Any]) -> bool:
+    def check_exchange(self, config: Dict[str, Any], check_for_bad: bool = True) -> bool:
```

Comment:
what is `check_for_bad` good for? i don't see it used / changed anywyere ... ?

Reply:
For scripts (like download, etc.) we may allow to run it for "bad" exchanges, allowing to download (or try to download) data when the main freqtrade logic will fail.
The scripts, thus, should call it with `check_for_bad=False`.

Reply:
The idea was to not print this message for the scripts. Maybe it's redundant since even download will fail for some of the 'bad' exchanges. So, what's the consensus -- let's remove this parameter?

Reply:
ah works for me - just asking as i didn't see it used anywhere :+1: